assets/js/init.js

```diff
@@ -72,6 +72,8 @@ const x = {
       const threshold = document.querySelector('header .cta').clientHeight;
       const content = document.getElementById('content');
       const padding = header.offsetHeight - threshold;
+      const toggler = document.getElementById('navbar-toggler');
+      const nav     = document.getElementsByClassName('nav')[0];
```

Comment:
Don’t pad like this. And don’t use `getElementsByClassName()` - add an id to the element instead